pkg/cmd/server/api/v1/types.go

```diff
@@ -73,6 +73,9 @@ const (
 	ControllersAll = "*"
 )
 
+// DisabledFeatures allows to turn off specific OpenShift components
+type DisabledFeatures []string
```

Comment:
Make this match the internal types.  That will allow you to drop the conversions and it keeps the code in sync.